hawkbit-core/src/test/java/org/eclipse/hawkbit/api/URLPlaceholderTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ *  Copyright (c) 2022 Bosch.IO GmbH and others.
+ *
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ */
+package org.eclipse.hawkbit.api;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+
+@Feature("Unit Tests - Artifact URL Handler")
+@Story("URL placeholder tests")
+class URLPlaceholderTest {
+
+    private URLPlaceholder.SoftwareData softwareData;
+    private URLPlaceholder placeholder;
+
+    public URLPlaceholderTest() {
+        this.softwareData = new URLPlaceholder.SoftwareData(1L, "file.txt", 123L, "someHash123");
+        this.placeholder = new URLPlaceholder("SuperCorp", 123L, "Super-1", 1L, softwareData);
+    }
+
+    @Test
+    @Description("Same object should be equal")
+    // Exception squid:S5785 - JUnit assertTrue/assertFalse should be simplified to
+    // the corresponding dedicated assertion
+    // Need to test the equals method and need to bypass magic logic in utility
+    // classes
+    @SuppressWarnings({ "squid:S5785" })
+    void sameObjectShouldBeEqual() {
+        assertTrue(softwareData.equals(softwareData));
```

Comment:
In fact thats exectly what I tried to avoid as the `isEqualsTo` method does already checks like the object-equals before calling the real equals method.

Reply:
As you see fit, what I wanted here is to use assertJ instead of Jupiter assertions to stick to one format across the tests. Would you be able to achieve the same with `assertThat(softwareData.equals(softwareData)).isTrue()`?